test/daml/Test/Lifecycle.daml

```diff
@@ -98,6 +98,8 @@ testSettle = script do
           . apoM (Lifecycle.lifecycle' observe25)
           . (1.0, )
 
+  let and x y = And x y []
```

Comment:
The new 'at-least 2' `And` datatype added in 28429cd makes it impossible to write 

```
x `And` y 
```
which is why I added this helper function. That is pretty annoying. Not sure if it should be part of the library. We could also consider re-introducing lower-case constructors like we had way back when i.e. 

```haskell
one : Claim
when : Inequality -> Claim -> Claim
and : Claim -> Claim -> Claim
```
and make `and` flatten nested `And`s as you go along.
On one hand that would affect all dependent applications. On the other hand, it would maintain the illusion of having nested `And` nodes. i.e. you would still write
```
a `and` b `and` c
```
in other words, it would just require lowercasing all combinators.

Reply:
A downside we previously had with these constructors was the clash with `Prelude.and`. That was pretty annoying too.